index.bs

```diff
@@ -5735,15 +5611,12 @@ partial interface MLGraphBuilder {
   <div class=algorithm-steps>
     1. If |splits| is an {{unsigned long}}, and |input|'s [=MLOperand/shape=][|options|.{{MLSplitOptions/axis}}] % |splits| is not 0, then [=exception/throw=] a {{TypeError}}.
     1. If |splits| is a sequence of {{unsigned long}}, and the sum of its elements is not equal to |input|'s [=MLOperand/shape=][|options|.{{MLSplitOptions/axis}}], then [=exception/throw=] a {{TypeError}}.
-    1. If any of the following sub-steps fail, [=exception/throw=] an "{{OperationError}}" {{DOMException}}.
+    1. *Make graph connections:*
         1. Let |output| be the result of [=copying an MLOperand=] given |input|.
-        1. Make a request to the underlying platform to:
-            1. Let |opImpl| be [=platform operator=] for the split operation, given |splits| and |options|.
-            1. Set |output|.{{MLOperand/[[operator]]}} to |opImpl|.
-            1. Create an [=platform operand=] |outputImpl| to represent the output, given |output| and |opImpl|.
-            1. Set |output|.{{MLOperand/[[operand]]}} to |outputImpl|.
-        1. Connect |input|.{{MLOperand/[[operand]]}} as input to |opImpl|.
-        1. Connect |output|.{{MLOperand/[[operand]]}} as output to |opImpl|.
+        1. Let |operator| be an [=operator=] for the split operation, given |splits| and |options|.
+        1. Set |output|.{{MLOperand/[[operator]]}} to |operator|.
+        1. Set |operator|'s [=operator/input=] to |input|.
+        1. Set |operator|'s [=operator/output=] to |output|.
```

Comment:
Same here, multiple outputs may need to be added into operator's outputs.

Reply:
Done